html/index.html

```diff
@@ -78,59 +82,70 @@ <h2>Software developer</h2>
       <div class="works_head">
         <h2 class="work_title">MY RECENT WORK</h2>
         <!--divider-->
-        <img alt="divider vector" src="../img/Vector.svg" class="divider_c">
+        <div class="divider_c"></div>
       </div>
+
       <div class="works_container">
+        <!-- reverse stands for the flex direction-->
         <div class="project">
           <div class="work_img"></div>
-          <h3 class="works_titles">Multi-Post Stories</h3>
-          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
-             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-          <ul class="work_tools_tags">
-            <li class="tool_tag">css</li>
-            <li class="tool_tag">html</li>
-            <li class="tool_tag">bootstrap</li>
-            <li class="tool_tag">Ruby</li>
-          </ul>
-            <button type="button" class="project_btn">See project</button>
+          <div class="sup_works">
+            <h3 class="works_titles">Multi-Post Stories</h3>
+            <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+               standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="work_tools_tags">
+              <li class="tool_tag">css</li>
+              <li class="tool_tag">html</li>
+              <li class="tool_tag">bootstrap</li>
+              <li class="tool_tag">Ruby</li>
+            </ul>
+              <button type="button" class="project_btn">See project</button>
```

Comment:
- As it is a requirement for this project and required skill for a front-end dev, sticking to the design as much as possible, please kindly fix the implementation of the `See project` buttons to match the design. The text is overlapping, see screenshots below and kindly fix that. 

***The design***
![theD](https://user-images.githubusercontent.com/62849872/150657080-bc0191a5-cf02-4562-9d0c-9f8a1c4ddd32.png)

***Your Implementation***
![yourI](https://user-images.githubusercontent.com/62849872/150657093-0deb9c3c-e2ca-4433-8296-34432f0e48c3.png)

Reply:
[OPTIONAL]
- Kindly remove the border from the `see project` button to make your design better.